src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -313,6 +313,16 @@ public XmlDocument UpdatePackageSources(XmlDocument nugetConfig, HashSet<string>
                             continue;
                         }
                     }
+                    // Remove the clear element wherever it is.
+                    // It will be added when we add the maestro managed sources.
+                    else if (currentNode.Name.Equals(VersionFiles.ClearElement, StringComparison.OrdinalIgnoreCase))
+                    {
+                        var nextNodeToWalk = currentNode.NextSibling;
```

Comment:
[nit] we have this set of instructions three times. Should we extract this to a helper?

Reply:
Done